main.go

```diff
@@ -132,6 +133,7 @@ func argsParse(args []string) (runOptionsType, error) {
 			*certKey,
 			*serverCert,
 			false,
+			*skipVerify,
```

Comment:
I suggest using labels here, there are quite a few entries, including two bools right next to each other. If we somehow mess up the order later on, the compiler won't warn you as is.